rosbag2_py/CMakeLists.txt

```diff
@@ -67,21 +67,21 @@ if(BUILD_TESTING)
   if(WIN32 AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     set(_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE_DEBUG}")
   endif()
-  set(pythonpath "${CMAKE_CURRENT_BINARY_DIR}/rosbag2_py;${CMAKE_CURRENT_SOURCE_DIR}")
-  if(NOT WIN32)
-    string(REPLACE ";" ":" pythonpath "${pythonpath}")
```

Comment:
Is the `CMAKE_CURRENT_BINARY_DIR;` part still needed for Windows?

Reply:
That's a good question, I'm trying to figure out the issue on Windows.
The path `${CMAKE_CURRENT_BINARY_DIR}/rosbag2_py` doesn't exist on Linux, thus why I deleted it.

Reply:
This wasn't needed on Windows finally.